privacy-config/privacy-config-impl/src/main/java/com/duckduckgo/privacy/config/impl/features/drm/RealDrm.kt

```diff
@@ -40,25 +38,12 @@ class RealDrm @Inject constructor(
     private val unprotectedTemporary: UnprotectedTemporary,
 ) : Drm {
 
-    override fun getDrmPermissionsForRequest(
-        url: String,
-        resources: Array<String>,
-    ): Array<String> {
-        val perms = mutableSetOf<String>()
-        if (shouldEnableDrmForUri(url.toUri())) {
-            resources.find { (it == PermissionRequest.RESOURCE_PROTECTED_MEDIA_ID) }?.let {
-                perms.add(PermissionRequest.RESOURCE_PROTECTED_MEDIA_ID)
-            }
-        }
-        return perms.toTypedArray()
-    }
-
-    private fun shouldEnableDrmForUri(uri: Uri): Boolean {
+    override fun isDrmAllowedForUrl(url: String): Boolean {
```

Comment:
nit: you could pass a uri instead which is what you are getting from `onPermissionRequest` and save a few parsings here and there.

Reply:
Good point. I tried going this route but it required changing gradle dependencies in `privacy-config-api`, so decided it wasn't worth it since we're likely to remove this anyway once we implement the consent prompt.